### PR TITLE
Graph

### PR DESCRIPTION
**Pull request by vritant24**
Added Google Charts and form inputs for the sidebar. 
Created a sidebar for the Map

### CODE REVIEW

File: public/app.js [hunk 1]
```diff
@@ -1,5 +1,11 @@
 var map;
 
+//Function runs on window loading
+window.onload = function() {
+  initGraph();
+  initMap();
```
Comment (drewatk) [inline public/app.js]: This shouldn't need calling

Comment (vritant24) [inline public/app.js]: It didn't work without calling. JS console tells me 'initMap not found'

Comment (drewatk) [inline public/app.js]: Go ahead and merge and I'll fix it later

Comment (vritant24) [inline public/app.js]: Cool, I'll let Jonah approve first tho

**Pull request merged**